keyboards/alice/README.md

```diff
@@ -0,0 +1,60 @@
+# TGR Alice
```

Comment:
Could this file be renamed to be lower case. 

Since this is cross platform, and some OS' have issues with casing, we try to keep all the files lowercase, to avoid weird issues.